docs/frameworks/nitrojs.md

```diff
@@ -0,0 +1,209 @@
+
+<Tabs groupId="packages">
+  <TabItem className="tab-item" value="npm" label="npm">
+      <div id="step1-install-npm">
+      ```
+      npm install genezio -g
+      ```
+      </div>
+  </TabItem>
+  <TabItem className="tab-item" value="pnpm" label="pnpm">
+      <div id="step1-install-pnpm">
+      ```
+      pnpm add -g genezio
+      ```
+      </div>
+  </TabItem>
+  <TabItem  className="tab-item" value="yarn" label="yarn">
+      <div id="step1-install-yarn">
+      ```
+      yarn add global genezio
+      ```
+      </div>
+  </TabItem>
+</Tabs>
+
+### 2. Ensure you have an Nitro.js App
+
+If you don't have an Express.js app, you can create one using the following steps:
+
+<details>
+  <summary>**Create a basic Nitro.js App**</summary>
+    
+### 1. Initialize a New Nitro.js Project
+
+Run the following command to create a starter template:
+
+<Tabs groupId="packages">
+  <TabItem className="tab-item" value="npm" label="npm">
+      <div id="init-project-npm">
+      ```
+      npx giget@latest nitro nitro-app --install
+      ```
+      </div>
+  </TabItem>
+  
+  <TabItem className="tab-item" value="pnpm" label="pnpm">
+      <div id="init-project-pnpm">
+      ```
+      pnpm dlx giget@latest nitro nitro-app --install
+      ```
+      </div>
+  </TabItem>
+  
+  <TabItem  className="tab-item" value="yarn" label="yarn">
+      <div id="init-project-yarn">
+      ```
+      yarn dlx giget@latest nitro nitro-app --install
+      ```
+      </div>
+  </TabItem>
+</Tabs>
+
+```bash
+cd nitro-app
+```
+### 2. Create a new route
+
+Next, navigate to the `server` directory, and create a new `users.ts` file.
+
+```typescript title="users.ts"
+export default defineEventHandler((event) => {
+  return [
+    { id: 1, name: "Alice" },
+    { id: 2, name: "Bob" },
+  ];
+});
+```
+### 3. Test the Nitro.js App
+
+Run the following command to start the Nitro.js app:
+<Tabs groupId="packages">
+  <TabItem className="tab-item" value="npm" label="npm">
+      <div id="start-app-npm">
+      ```
+      npm run dev
+      ```
+      </div>
+  </TabItem>
+  
+  <TabItem className="tab-item" value="pnpm" label="pnpm">
+      <div id="start-app-pnpm">
+      ```
+      pnpm dev
+      ```
+      </div>
+  </TabItem>
+  <TabItem  className="tab-item" value="yarn" label="yarn">
+      <div id="start-app-yarn">
+      ```
+      yarn dev
+      ```
+      </div>
+  </TabItem>
+</Tabs>
+Open a web browser and navigate to [http://localhost:8080](http://localhost:8080) to see the app running.
+Navigate to [http://localhost:8080/users](http://localhost:8080/users) to see the newly created route.
+</details>
+
+## Deployment Guide
+
+## 1. Build your app for deployment
+```
+nitro build --preset genezio
+```
+## 2. Personalize your generated `genezio.yaml` file
+By building your nitro app using the Genezio preset, a default `genezio.yaml` file will be Generated, which can be further adjusted for specific cases.
```

Comment:
maybe we add: "a default file will be generated **if you don't already have one**"